### DISCUSSION TITLE
Error: Atri is not found in the directory

### DISCUSSION THREAD

**Discussion by sambit-sankalp** [Atri-Labs/atrilabs-engine#583] [Help: Installation & Start] [answered]
![doubt](https://user-images.githubusercontent.com/82284130/208250806-ee3a0196-3ccc-41d0-8454-c564fdc23748.png)
This is the issue I am facing while running `atri start` in the directory.

**Comment by darshitac11** [comment, marked as answer]
@sambit-sankalp - It doesn't look like you have installed atri. 

Please follow the steps provided [here](https://docs.atrilabs.com/getting-started/installation/install_with_pipenv). If you still receive the same error, please submit a detailed report of the commands that resulted in the above error.

**Comment by sambit-sankalp** [comment]
@darshitac11 
![image](https://user-images.githubusercontent.com/82284130/208253919-1feca8b8-f857-46fc-a9c4-304bff5946c5.png)
This was the output when I installed atri using pipenv.

Is atri not installed?

**Reply by cruxcode** [reply]
There is something weird with how Python is setup in your machine. The virtual environment is using python 3.11 while you have libraries from 3.9. Please tell us what is the exact version of the python installed in your system. To find out this, you can run `python3 --version`.

**Reply by sambit-sankalp** [reply]
I removed one version of python and tried installing but still facing issues.
The error: 
![image](https://user-images.githubusercontent.com/82284130/208313615-3d38bba6-c17c-42e7-ac34-aafcc39b89cc.png)

**Reply by cruxcode** [reply]
I don't understand why `pipenv` is creating a virtual environment twice. Can you please check if the `pipenv` is of latest version?
